client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -24,9 +25,7 @@ type Template = VerticalsTemplates.Template;
 const VERTICALS_TEMPLATES_STORE = VerticalsTemplates.register();
 
 const DesignSelector: FunctionComponent = () => {
-	const siteVertical = useSelect(
-		select => select( 'automattic/onboard' ).getState().siteVertical
-	);
+	const siteVertical = useSelect( select => select( ONBOARD_STORE ).getState().siteVertical );
```

Comment:
I'm not sure why we didn't do destructuring assignment here (not introduced here). Can we combine the selectors and remove the one below?

```suggestion
	const { selectedDesign, siteVertical } = useSelect( select => select( ONBOARD_STORE ).getState() );
```